dle/entrypoint

```diff
@@ -22,9 +23,21 @@ if [[ ($INIT_SUPERUSER) && ("$INIT_SUPERUSER" = "True") ]]; then
   # Uses the DJANGO_SUPERUSER_* environment variables to create a superuser
   # TODO find a way to only run createsuperuser if it doesn't already exist
   echo "Creating superuser"
-  python3 manage.py createsuperuser --noinput
+  python3 manage.py create_superuser_if_none_exists --user $DJANGO_SUPERUSER_USERNAME --email $DJANGO_SUPERUSER_EMAIL --password $DJANGO_SUPERUSER_PASSWORD
 fi
 
+
+# Load fixtures, a JSON file containing the data for the database
+# Place the JSON file in /app/data/fixtures (from S3 unless we switch to Git LFS)
+if [[ ($LOAD_FIXTURES) && ("$LOAD_FIXTURES" = "True") ]]; then
+  echo "Loading fixtures"
+  python3 manage.py loaddata /app/data/fixtures/*.json
+  echo "Finished loading fixtures"
+fi
+
+
+# Otherwise, if not loading fixtures, you can run a scrape for new data
+# This will take a long time
 if [[ ($LOAD) && ("$LOAD" = "True") ]]; then
```

Comment:
Add HC as well?

Reply:
Already in `main`, I started this PR before the HC script got merged, so should be good to go